.npmignore

```diff
@@ -1,22 +1,6 @@
-.idea
-.github
-docs
-example
-src
-test
-.babelrc
-.editorconfig
-.eslintignore
-.eslintrc
-.git
-.gitmodules
-.jshintrc
-.postcssrc.yml
-.stylelintrc
-CODEOWNERS
-cypress.json
-tsconfig.json
-tslint.json
-webpack.config.js
-yarn.lock
-devserver.js
+*
+!dist/**/*
+!types/**/*
+!LICENSE
+!README.md
```

Comment:
readme contains some pictures from the `assets` as well

Reply:
Do we need it in the package? We need to use S3 for images then